src/client/components/household-information/AnnualIncomeSection.jsx

```diff
@@ -43,15 +43,15 @@ class AnnualIncomeSection extends React.Component {
             <table className="review usa-table-borderless">
               <tbody>
                 <tr>
-                  <td>Children Gross Income:</td>
+                  <td>Gross Annual Income from Employment:</td>
```

Comment:
Do we want to remove `Child` from the beginning? Seems like that could be useful in being super clear about what each field maps to